packages/bff/src/services/agreementService.ts

```diff
@@ -349,3 +459,26 @@ export function getCurrentDescriptor(
   }
   return descriptor;
 }
+
+function assertContentMediaType(
+  contentType: string,
+  agreementId: string,
+  documentId: string
+): void {
+  // Content types from: https://doc.akka.io/api/akka-http/current/akka/http/scaladsl/model/ContentTypes
+  const contentTypes = [
+    "NoContentType",
+    "application/grpc+proto",
+    "application/json",
+    "application/octet-stream",
+    "application/x-www-form-urlencoded",
+    "text/csv(UTF-8)",
+    "text/html(UTF-8)",
+    "text/plain(UTF-8)",
+    "text/xml(UTF-8)",
+  ];
```

Comment:
ℹ️  in the context of [PR](https://github.com/pagopa/interop-be-monorepo/pull/940) [was added](https://github.com/pagopa/interop-be-monorepo/pull/940/files#diff-51d4336b95699e2593edc1226d8d04b82d8239ddd219a07c32b593a0d6bac44bR42) [package `mime` ](https://github.com/broofa/mime). 
In your case may be useful to replace the hardcoded string with mime types const from this lib?

Reply:
I noticed that there is already a duplicate of this constant (https://github.com/pagopa/interop-be-monorepo/blob/0cba067d46d70529a0ff34400e929216fbeaecae/packages/bff/src/services/purposeService.ts#L422) done by @MalpenZibo . Should we extract into a common constant inside the `bff`?

Maybe also using `mime` instaed of hardcoding them

Reply:
As it turns out, `mime` does not contain consts or types for the types but rather an API to see if a type/extension is valid or not.

I am not sure about using it, as it can validate types that we do not want. But, in Scala they do a similar thing simply using the standard Mime types of the web framework. Should we stick with the common magic strings or should we use the `mime` api to validate?